stac-example/src/main/resources/application.conf

```diff
@@ -377,6 +377,114 @@ layers = {
         styles = [{ type = "rgbstyleconf" }]
     }
 
+    lc8-red-mixed = {
+        type = "rastersourceconf"
+        name = "lc8-red-mixed",
+        title = "lc8-red-mixed"
+        source = "gdal+https://s3-us-west-2.amazonaws.com/landsat-pds/c1/L8/015/032/LC08_L1TP_015032_20180521_20180605_01_T1/LC08_L1TP_015032_20180521_20180605_01_T1_B4.TIF"
+
+        default-style = "red-to-blue"
+        styles = [
+            {
+                name = "red-to-blue"
+                title = "Red To Blue"
+                type = "colorrampconf"
+                colors = ${color-ramps.red-to-blue}
+                stops = 64
+            }
+        ]
+    }
+
+    stac-mixed-lc8-rgb = {
+        type = "mapalgebrasourceconf"
+        name = "stac-mixed-lc8-rgb"
+        title = "Landsat LayerUS RGB"
+        algebra = {
+            "args": [
+                {
+                    "args": [
+                        {
+                            "args": [
+                                {
+                                    "args": [
+                                        {
+                                            "name": "lc8-red-mixed",
```

Comment:
That's the non STAC layer used in the MapAlgebraSource. The rest of layers are STAC Layers.